gallery/src/components/NxBinaryDonutChart/NxBinaryDonutChartPage.tsx

```diff
@@ -56,10 +56,31 @@ const NxBinaryDonutChartPage = () =>
           <NxTable.Row>
             <NxTable.Cell>percent</NxTable.Cell>
             <NxTable.Cell>number</NxTable.Cell>
-            <NxTable.Cell>Yes</NxTable.Cell>
+            <NxTable.Cell>No</NxTable.Cell>
             <NxTable.Cell>
               Percentage which this donut represents. E.g. when 0 the donut is empty, and as it increases towards 100
               the amount of the donut which is filled in increases.
+              <NxWarningAlert>
+                Deprecated: With the introduction of the <NxCode>value</NxCode> prop, you can now define
+                current value, and the percent will be calculated based on the <NxCode>value</NxCode> prop
+                and the <NxCode>maxVal</NxCode> prop.
+              </NxWarningAlert>
+            </NxTable.Cell>
+          </NxTable.Row>
+          <NxTable.Row>
```

Comment:
Let's have this rows above the `percent` row so we aren't leading with the deprecated one

Reply:
This is updated here b089e64cd333e786b70425b84c1f0e8972797b7e.